src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -135,6 +135,11 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
         $this->objectClassResolver = $objectClassResolver;
     }
 
+    public function getSupportedTypes(?string $format): ?array
+    {
+        return null;
```

Comment:
same note as on AbstractNormalizer, unless I miss something?

Reply:
no, you are right 👍